components/drops/DropCard.vue

```diff
@@ -130,10 +131,7 @@ onMounted(async () => {
 .drop-card {
   &:hover {
     .drop-card-banner {
```

Comment:
this does not look good as `&-banner` is defined underneath, I would add `&:hover` inside it or use class group hover

otherwise ok

Reply:
There is detail that when hovering on the whole card `drop-card`,  the style should change. That's why I could not add `&:hover` inside only the banner `drop-card-banner`.  wdyt?

Reply:
I would consider refactoring to something cleaner since I find it kinda hard to read

Reply:
lmk if you have any idea to make it with a better solution.